app.js

```diff
@@ -208,6 +208,15 @@ app.use((err, req, res, next) => {
         return res.status(403).render('500.badToken.njk');
     }
 
+    // express-openid-connect error.
+    if (err.name === 'AggregateError') {
+        if (/^Issuer.discover\(\) failed/.test(err.message)) {
+            return res.status(500).render('oidc-provider-unreachable.njk', {
+                backTarget: req?.session?.referrer
+            });
```

Comment:
Will you not need to pass in the nonce here?

Reply:
it is added to the [`res.locals` object here](https://github.com/CriminalInjuriesCompensationAuthority/cica-web/blob/feature/cica-691/one-login-outage/app.js#L110). This object is passed implicitly to the `res.render` function. No need to manually do it.

Side note: Yet anther reason to decommission the `templateEngine` render function